spec/classes/python_spec.rb

```diff
@@ -31,6 +31,18 @@
             end
           end
 
+          describe 'with python::manage_python_package' do
```

Comment:
can you also add a test that explicitly sets the new parameters to false and check if the catalog still compiles and the packages won't be in it?